app/controllers/homepage_controller.rb

```diff
@@ -3,6 +3,10 @@ class HomepageController < ApplicationController
 
   def index
     homes = Home.all
-    @last_three = homes.last(3)
```

Comment:
I think this would be better:


```
def index
  @recent_homes = Home.last(3)
end
```

Then update the view (see other comment below)